public/components/datasources/components/manage/accelerations/acceleration_details_flyout.tsx

```diff
@@ -24,20 +24,90 @@ import {
   onRefreshButtonClick,
   onDiscoverButtonClick,
   onDeleteButtonClick,
-} from '../accelerations/helpers/utils';
+} from './utils/acceleration_utils';
+import { coreRefs } from '../../../../../framework/core_refs';
+import { OpenSearchDashboardsResponse } from '../../../../../../../../src/core/server/http/router';
 
 export interface AccelerationDetailsFlyoutProps {
   acceleration: any;
```

Comment:
will fix this

Reply:
coming with the follow up fix on this issue: https://github.com/opensearch-project/dashboards-observability/issues/1534